dandiapi/api/auth.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework.authentication import TokenAuthentication
```

Comment:
note  that this is a new file.  If you would like to see it somewhere else -- let me know , but it could not go under dandi.api.views.auth due to then circular import.